### PR TITLE
Use RSS feed libraries to mangle the RSS Feed

### PR DESCRIPTION
**Pull request by ifosch**
Using XML libraries to manipulate RSS feed is too complicated.
This implementation uses
[gofeed](https://github.com/mmcdole/gofeed) to read current
feed, and [podcast](https://github.com/eduncan911/podcast) to
add the new episode and write it to stdout.

This fixes the bug on newlines in episode description (#44),
while making the code easier to understand.

### CODE REVIEW

**Comment by ifosch** [comment]
Fixes #44, but this requires #56 to be merged first.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# appu
+
+Appu is a toolkit for podcast edition and publishing.
```
Comment (edubxb) [inline README.md:3]: **A**utomatic **P**odcat **PU**blisher, aka appu, is a toolkit for podcast edition and publishing.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Usage
+
+### Requirements
+
+You'll need the following:
+- Docker, for Windows and Mac, you should use Docker Desktop.
+- Publishing infrastructure based on an online storage, and a CDN with invalidation features.
+  Currently only AWS S3 and CloudFront are supported.
+  You'll need credentials for uploading the episodes' audio files and the RSS feed and then invalidating them on the CDN.
```
Comment (edubxb) [inline README.md:25]: You'll need the following:
- Docker, for Windows and Mac, you should use Docker Desktop.
- Publishing infrastructure based on an online storage, and a CDN with invalidation features (currently only AWS S3 and CloudFront are supported).
- Valid credentials for uploading the episodes' audio files and the RSS feed, and then invalidating them on the CDN.

**Review by edubxb — APPROVED** [review]
LGTM, but see my comments regarding the some documentation changes.

**Pull request merged**